### PR TITLE
No passcode at campaign creation

### PR DESCRIPTION
**Pull request by mauditecandela**
- Adds checkbox if the instructor wants to set the no passcode as default

- Overrides the course if the default_passcode is "no-passcode" as ''
- Adds locales for the translations

### CODE REVIEW

File: lib/course_creation_manager.rb [hunk 1]
```diff
@@ -80,6 +80,9 @@ def duplicate_slug?
   end
 
   def set_passcode
+    if @course_params[:passcode] = 'no-passcode'
```
Comment (ragesoss) [inline lib/course_creation_manager.rb]: Should be `==` ?

**Comment by ragesoss** [comment]
This looks okay to me. We should include a test for it.

**Comment by ragesoss** [comment]
Probably a new spec in `courses_controller_spec` will be the easiest way to do this.

**Comment by mauditecandela** [comment]
@ragesoss this took me a while as the Haml syntax is still a bit tricky for me 😅 
If the user checks the "no passcode" checkbox, then will override the passcode either it's set or it's randomnly generated by the course_creation_manager.
I would prefer to hide the passcode field if the user checks the no passcode checkbox in order to avoid misunderstandings but I didn't manage to do it. 
Indeed, we need a test,  I will work on it tomorrow.

**Pull request merged**